en-US/D.xml

```diff
@@ -189,7 +189,13 @@
 			<term>dialog box</term>
 			 <listitem>
 				<para>
-					Refer to the Word Usage chapter of the <citetitle>IBM Style Guide</citetitle> for usage information related to this and similar terms.
+					<emphasis>n.</emphasis> Use "dialog box" (<emphasis>not</emphasis> "dialog", "dialogue", "dialogue box", or "window"), to refer to an element that is displayed for a user to interact with a graphical user interface.
```

Comment:
I suggest also adding "pop up" and/or "pop-up window" to the "do not use" list.

Reply:
"pop-up" is already there, but maybe you want to elaborate the entry?

Reply:
I've added to the "pop-up" entry.